### PR TITLE
proto: allow unrestricted length character length in SVCB param values

### PR DESCRIPTION
**Pull request by djc**
Fixes yet another fuzzing issue.

### CODE REVIEW

File: crates/proto/src/rr/rdata/svcb.rs [hunk 1]
```diff
@@ -1051,6 +1054,7 @@ impl fmt::Display for Unknown {
 /// ```
 pub fn read(decoder: &mut BinDecoder<'_>, rdata_length: Restrict<u16>) -> ProtoResult<SVCB> {
     let start_index = decoder.index();
+    println!("{:?}", start_index);
```
Comment (bluejekyll) [inline crates/proto/src/rr/rdata/svcb.rs:1057]: these prtinln's look like debug statements?

**Review by bluejekyll — CHANGES_REQUESTED** [review]
This looks good to me, but the println's look like some debug logic that got left behind.

Thanks for working on all of this!

File: crates/proto/src/rr/rdata/svcb.rs (hunk 1 above)
Comment (djc) [reply crates/proto/src/rr/rdata/svcb.rs]: Oops, sorry about that.

**Review by bluejekyll — APPROVED** [review]
Thanks for fixing this up, @djc.

**Pull request merged**